package.json

```diff
@@ -12,20 +12,21 @@
     "live streaming"
   ],
   "author": "Mux <help@mux.com> (https://mux.com/)",
-  "main": "./cjs/index.js",
+  "main": "./lib/mux.js",
   "exports": {
-    "import": "./esm/index.mjs",
-    "require": "./cjs/index.js"
+    "import": "./lib/mux.mjs",
+    "require": "./lib/mux.js"
   },
+  "files": [
```

Comment:
Hmm--packaging source files is (for me) intentional in a Node package, because having the source TypeScript as a sidecar is valuable. Can you explain your thinking here?